packages/block-editor/src/components/block-toolbar-link-control/settings-toolbar-item.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	DropdownMenu,
+	MenuGroup,
+	MenuItem,
+	ToolbarItem,
+} from '@wordpress/components';
+import {
+	chevronDown as arrowDownIcon,
+	check as checkIcon,
+} from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+import { useContext } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import ToolbarLinkControlContext from './context';
+
+export default function SettingsToolbarItem() {
+	const { currentLink, updateCurrentLink, setPreferredDropdown } = useContext(
+		ToolbarLinkControlContext
+	);
+	const { opensInNewTab, rel } = currentLink;
+	return (
+		<ToolbarItem>
+			{ ( toolbarItemProps ) => (
+				<DropdownMenu
+					popoverProps={ { position: 'bottom' } }
+					className="link-option"
+					contentClassName="link-options__popover"
+					icon={ arrowDownIcon }
+					onToggle={ ( isOpen ) => {
+						if ( isOpen ) {
+							setPreferredDropdown( 'settings' );
+						}
+					} }
+					toggleProps={ {
+						...toolbarItemProps,
+						name: 'link-options',
+						title: __( 'Link options' ),
```

Comment:
This renders a title attribute on the button. Title attributes are an accessibility and usability anti-pattern and should not be used. In the last years, a lot of work has been done to [remove as much title attributes as possible](https://core.trac.wordpress.org/query?keywords=~title-attribute) from the admin. For no reason new title attributes should be introduced.
Also, the button doesn't have a tooltip so it doesn't expose its name on focus. Looking at how the DropdownMenu toggle button works, I think a `label` prop needs to be used directly on the DropdownMenu to correctly label the button and make the tooltip work.
Lastly, not sure what the `name` prop does other than rendering a name attribute `name="link-options"` on the button. Is this prop necessary for some reason?

Reply:
I think this actually ends up setting the `title` prop on a `Button`, which in turn sets the `aria-label` of the underlying HTML `<button>`?

Reply:
That said, I haven't actually tested out this PR, so I could be wrong.

Reply:
I can confirm it is adding a `title` attribute  to the button element. If we want to render a tooltip, we should instead use `showTooltip` together with `label="Link options"`, which sets an `aria-label` and shows the tooltip as used in editor icon buttons.
I'm also not sure what that `name` prop is about; it's not a documented option in either the `DropdownMenu` or the `Button` components.